### DISCUSSION TITLE
Can't Replicate My Own Starship Configuration

### DISCUSSION THREAD

**Discussion by grantelledge** [starship/starship#5708] [Q&A]
Hi there! Feel like I'm going nuts here, I'm trying to replicate a Starship configuration I have on cmd on one device on another. The only clue I have to work from is that I get the following error when I start a new cmd prompt:

`C:\Users\...\AppData\Local\clink\starship.lua:1: attempt to call a nil value` (directory is correct, just reducing the shared information here at the `...`)

Like I said, replicating the exact working setup I have on another device, but for reference, here's the entirety of that Lua file:

```
load(io.popen('starship init cmd'):read("*a"))()
os.setenv('STARSHIP_CONFIG', 'C:\\Users\\...\\AppData\\Local\\.config\\starship.toml')
```

Clink and dependencies are installed and functioning, and I do have a Nerd Font installed. I have that `starship.toml` replicated from my other device as well, path is correct, etc. Thanks in advance for any help -- didn't want to submit this as a bug, as it's worked for me perfectly in the past!

**Comment by davidkna** [comment]
Is starship itself installed and accessible from the `$Path`?

**Reply by grantelledge** [reply]
Should've noted the install -- yes, installed with `winget`.

On `$Path`, I'm not seeing it on either the working device or the non-working one -- maybe I don't know what I should be looking for, though? Any help sincerely appreciated.

**Reply by davidkna** [reply]
The installer used by winget should already handle `$Path`. But a reboot or logging-in again may be required to propagate the change.

**Reply by grantelledge** [reply]
Unfortunately not. Any other recommendations? My understanding from your comment is that `$Path` (either globally or for my user) would need to include a route to where my `starship.lua` file is located, or is it somewhere else? This (or any other path) doesn't make a lot of sense to me given the error message, but I'm not at all an expert in this kind of thing! Thanks again.
